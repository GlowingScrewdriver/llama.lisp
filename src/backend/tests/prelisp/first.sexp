(c-lisp
    (define ((error_check void) (res int))
        (if (gt res 0)
            (call exit 1))
        (ret res))

    (define ((main int))
        (declare res ,cudevice)
        (set res ,(myError (call print 5)))
        (ret 0)))
