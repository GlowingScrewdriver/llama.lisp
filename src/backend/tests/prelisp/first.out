(c-lisp
  (define ((error_check void) (res int))
    (if (gt res 0) (call exit 1))
    (ret res))
  (define ((main int))
    (declare res int)
    (set res (call error (call print 5)))
    (ret 0)))
